boot-qemu.py

```diff
@@ -252,6 +293,93 @@ def supports_efi(self):
         return False
 
 
+class ARM64QEMURunner(QEMURunner):
+
+    def __init__(self):
+        super().__init__()
+
+        self.cmdline += ['console=ttyAMA0', 'earlycon']
+
+        self._default_kernel_path = Path('arch/arm64/boot/Image.gz')
+        self._initrd_arch = 'arm64'
+        self._qemu_arch = 'aarch64'
+
+    def _can_use_kvm(self):
+        return platform.machine() == 'aarch64' and self._have_dev_kvm_access()
+
+    def run(self):
+        machine = ['virt', 'gic-version=max']
+
+        if not self.use_kvm:
+            cpu = ['max']
+
+            self._set_qemu_path()
+            if (qemu_ver := self._get_qemu_ver_tuple()) >= (6, 2, 50):
```

Comment:
got a comment for what's special about 6.2.50?

Reply:
It is the two gitlab links in the bottom section, should I add a comment here like "See the two gitlab links below for more details"?

Reply:
yes please

Reply:
Done: 9cdf19f21bcded24591d66f39c1228a2929d628e